numba/core/ir_utils.py

```diff
@@ -637,8 +644,15 @@ def remove_dead_block(block, lives, call_table, arg_aliases, alias_map,
         # let external calls handle stmt if type matches
         if type(stmt) in remove_dead_extensions:
             f = remove_dead_extensions[type(stmt)]
-            stmt = f(stmt, lives, lives_n_aliases, arg_aliases, alias_map, func_ir,
-                     typemap)
+            stmt = f(
+                stmt,
+                lives,
+                lives_n_aliases,
+                arg_aliases,
+                alias_map,
+                func_ir,
+                typemap,
+            )
```

Comment:
Does this help with `flake8`? Would a line wrap suffice?

Reply:
See above comment - flake8 is happy to just have a line wrap, this is done for compliance with Black which IMO is more readable.